cb-talent-development/employer-style-base
# CE-500: Global Nav Updates

**Purpose**
> As a user, I want to see a CareerBuilder header and navigation style that matches the rest of the brand ecosystem (job seeker side and logged in product user) so my experience is consistent wherever I am.

This PR updates the styling of our global header and navigation to better follow CB's other sites. Similar to the header-nav seen here: https://recruitment.careerbuilder.com/candidates/candidates-list

More specifically, this PR just updates variables controlling the navigation in our Style Base.

**JIRA**
https://careerbuilder.atlassian.net/browse/CE-500

**Changes**
* Improvements and fixes
  * Our navigation now matches styling seen with CB's other sites.

* Changes to developer setup/environment
  * N/A

* Architectural changes
  * N/A

* Migrations or Steps to Take on Production
  * N/A
  
* Library changes
  * N/A

* Side effects
  * The subnav in this PR is out of line with the styling updates. This however is to be fixed in another story @toastercup worked on.
  * There may be some side effects with the mobile menu, however, I did what I could to make sure that styles for the mobile menu are consistent with what is seen on desktop.
  * The the new descriptor that is seen next to the logo goes away around 640px because the header is too crowded (talked with Stephanie already)
  * Some colors are not exact with what is seen in the brand ecosystem. This was discussed with Stephanie and we agreed to stick with what we have in Style Base
  * Search was temporarily pulled out as it will be worked on as a drawer in another story. Am currently discussing with Stephanie on the best approach for this.

**Screenshots**
* Before
![screen shot 2017-11-14 at 5 19 18 pm](https://user-images.githubusercontent.com/5348098/32850470-e2e2bce6-c9f7-11e7-86ce-a3c2bbbaab61.png)

![screen shot 2017-11-14 at 5 19 27 pm 2](https://user-images.githubusercontent.com/5348098/32850478-e6ba3146-c9f7-11e7-9bc4-35d39dfed591.png)

![screen shot 2017-11-14 at 5 20 42 pm 2](https://user-images.githubusercontent.com/5348098/32850489-ee0b15e6-c9f7-11e7-8bc5-ac9f87949ba6.png)

![screen shot 2017-11-14 at 5 21 16 pm 2](https://user-images.githubusercontent.com/5348098/32850500-f5e1da16-c9f7-11e7-8227-51e4505a8d91.png)

![screen shot 2017-11-14 at 5 21 47 pm 2](https://user-images.githubusercontent.com/5348098/32850505-fb37b6ac-c9f7-11e7-939b-d8e804003081.png)

* After
![screen shot 2017-11-14 at 5 22 39 pm](https://user-images.githubusercontent.com/5348098/32850518-04b5fa04-c9f8-11e7-8c79-a003dfd2ba9f.png)

![screen shot 2017-11-14 at 5 22 45 pm 2](https://user-images.githubusercontent.com/5348098/32850526-0a08ede0-c9f8-11e7-9db7-5b49706877d2.png)

![screen shot 2017-11-14 at 5 23 24 pm 2](https://user-images.githubusercontent.com/5348098/32850532-0dde9db6-c9f8-11e7-9976-fe5c7c4dce97.png)

![screen shot 2017-11-14 at 5 23 54 pm 2](https://user-images.githubusercontent.com/5348098/32850537-133cf938-c9f8-11e7-9c66-e35fb23979ac.png)

**Feature Server**
http://web.employer-3.development.c66.me/

**How to Verify These Changes**
* Specific pages to visit
  * Sign in and sign out states
  * Homepage
  * Any page with the buy box (/recruiting-solutions/post-jobs)
  * Any page with the subnav (/recruiting-solutions/small-business-subscription-plans-and-pricing)

* Steps to take
  * Comb through the ACs in the Jira and compare what you see with what is up on the server
  * Go here: https://recruitment.careerbuilder.com/candidates/candidates-list and compare and note any differences you come across
  * Resize down to mobile; "For Employers" should go away when you're in table view. The mobile menu should still function
  * Click on the "Request Demo" button in the menu, the form should appear in the modal. If there is something off with it's positioning, make note of this.
  * Make sure the Buy Box still functions correctly as you scroll
  * Make sure the Sub Nav still stickifies correctly as you scroll
  * Verify the links work correctly
  * Sign in to view the logged in state of the header and compare with the ACs in the Jira

* Responsive considerations
  * The mobile subnav was restyled slightly to compensate for the new updates.
  * Sign in and make sure the mobile subnav still looks ok

**Relevant PRs/Dependencies**
  * https://github.com/cbdr/employer/pull/1045

**Additional Information**
  * Although the images show the fonts with normal weight, the server was updated to use bold fonts instead per Stephanie. The Request Demo link also should not have a background color for its hover state and should just underline like a normal link.
